deeprank_gnn/models/grid.py

```diff
@@ -118,54 +127,69 @@ def features(self) -> Dict[str, numpy.array]:
         return self._features
 
     def add_feature_values(self, feature_name: str, data: numpy.ndarray):
-        """ Makes sure feature values per grid point get stored.
+        """Makes sure feature values per grid point get stored.
 
-            This method may be called repeatedly to add on to existing grid point values.
+        This method may be called repeatedly to add on to existing grid point values.
         """
 
         if feature_name not in self._features:
             self._features[feature_name] = data
         else:
             self._features[feature_name] += data
 
-    def _get_mapped_feature_gaussian(self, position: numpy.ndarray, value: float) -> numpy.ndarray:
+    def _get_mapped_feature_gaussian(
+        self, position: numpy.ndarray, value: float
+    ) -> numpy.ndarray:
 
         beta = 1.0
 
         fx, fy, fz = position
-        distances = numpy.sqrt((self.xgrid - fx) ** 2 + (self.ygrid - fy) ** 2 + (self.zgrid - fz) ** 2)
+        distances = numpy.sqrt(
+            (self.xgrid - fx) ** 2 + (self.ygrid - fy) ** 2 + (self.zgrid - fz) ** 2
+        )
 
         return value * numpy.exp(-beta * distances)
 
-
-    def _get_mapped_feature_fast_gaussian(self, position: numpy.ndarray, value: float) -> numpy.ndarray:
+    def _get_mapped_feature_fast_gaussian(
+        self, position: numpy.ndarray, value: float
+    ) -> numpy.ndarray:
 
         beta = 1.0
         cutoff = 5.0 * beta
 
         fx, fy, fz = position
-        distances = numpy.sqrt((self.xgrid - fx) ** 2 + (self.ygrid - fy) ** 2 + (self.zgrid - fz) ** 2)
+        distances = numpy.sqrt(
+            (self.xgrid - fx) ** 2 + (self.ygrid - fy) ** 2 + (self.zgrid - fz) ** 2
+        )
 
         data = numpy.zeros(distances.shape)
 
-        data[distances < cutoff] = value * numpy.exp(-beta * distances[distances < cutoff])
+        data[distances < cutoff] = value * numpy.exp(
+            -beta * distances[distances < cutoff]
+        )
 
         return data
 
-    def _get_mapped_feature_bsp_line(self, position: numpy.ndarray, value: float) -> numpy.ndarray:
+    # def _get_mapped_feature_bsp_line(
+    #     self, position: numpy.ndarray, value: float
+    # ) -> numpy.ndarray:
 
-        order = 4
+    #     order = 4
 
-        fx, fy, fz = position
-        bsp_data = (bspline((self.xgrid - fx) / self.resolution, order) *
-                    bspline((self.ygrid - fy) / self.resolution, order) *
-                    bspline((self.zgrid - fz) / self.resolution, order))
+    #     fx, fy, fz = position
```

Comment:
I commented the BSP line everywhere since we did not know where bspline came from.

Reply:
I think I forgot the

**from scipy.signal import bspline**

Reply:
Ok I'll insert it